src/converters/collection-converter.js

```diff
@@ -0,0 +1,33 @@
+import Collection from '../model/collection';
+
+const collectionConverter = (thing, conversionHandler) => {
+	const id = thing.id;
+	const type = thing.type;
+	const {
+		view: {first, last, next},
+		totalItems,
+		numberOfItems,
+		member,
+	} = thing.attributes;
+
+	let items = [];
+
+	if (member.length !== 0) {
+		items = member.map(item => conversionHandler.convert(item.thing));
```

Comment:
I prefer doing affirmative ifs, that way you don't have to negate the conditional in your head

Reply:
Also ternaries you could do just the map right?